test/CMakeLists.txt

```diff
@@ -258,6 +263,28 @@ if(UMF_BUILD_GPU_TESTS AND UMF_BUILD_LEVEL_ZERO_PROVIDER)
                                PRIVATE ${LEVEL_ZERO_INCLUDE_DIRS})
 endif()
 
+if(UMF_BUILD_GPU_TESTS AND UMF_BUILD_CUDA_PROVIDER)
```

Comment:
```
... AND CUDA_FOUND)
...
else()
    message("Skipping CUDA example, since libcuda was not found")
...
```

I got the following error now:
```
[ 93%] Linking CXX executable umf_example_cuda_shared_memory
/usr/bin/ld: cannot find -lcuda: No such file or directory
collect2: error: ld returned 1 exit status
make[2]: *** [examples/CMakeFiles/umf_example_cuda_shared_memory.dir/build.make:99: examples/umf_example_cuda_shared_memory] Error 1
make[1]: *** [CMakeFiles/Makefile2:2562: examples/CMakeFiles/umf_example_cuda_shared_memory.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```